cdc/processor/pipeline/sink.go

```diff
@@ -98,14 +103,20 @@ func (n *sinkNode) CheckpointTs() model.Ts { return atomic.LoadUint64(&n.checkpo
 func (n *sinkNode) Status() TableStatus    { return n.status.Load() }
 
 func (n *sinkNode) Init(ctx pipeline.NodeContext) error {
-	// do nothing
+	n.replicaConfig = ctx.ChangefeedVars().Info.Config
+	return n.InitWithReplicaConfig(false, ctx.ChangefeedVars().Info.Config)
+}
+
+func (n *sinkNode) InitWithReplicaConfig(isTableActorMode bool, replicaConfig *config.ReplicaConfig) error {
+	n.replicaConfig = replicaConfig
+	n.isTableActorMode = isTableActorMode
 	return nil
 }
 
 // stop is called when sink receives a stop command or checkpointTs reaches targetTs.
 // In this method, the builtin table sink will be closed by calling `Close`, and
 // no more events can be sent to this sink node afterwards.
-func (n *sinkNode) stop(ctx pipeline.NodeContext) (err error) {
+func (n *sinkNode) stop(ctx context.Context) (err error) {
```

Comment:
May be it is better to add a debug log here to indicate which table is closed?

Reply:
yes, I will add it in actor PR